### DISCUSSION TITLE
Is Blitz production ready now ?

### DISCUSSION THREAD

**Discussion by danushk** [blitz-js/blitz#1794] [General]
title says it :D

**Comment by flybayer** [comment]
Well, we're officially in beta. That said, there are many, many blitz apps already in production. So sort of depends on you and whether you are ok with beta software or not.

You can see a partial list of some here in the wiki: https://github.com/blitz-js/blitz/wiki

Also, fyi we have quite a lot of changes happening right now that will be released within a week or two.
